skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2007 Zachary Pincus
+# This file is part of CellTool.
+#
+# CellTool is free software; you can redistribute it and/or modify
+# it under the terms of version 2 of the GNU General Public License as
+# published by the Free Software Foundation.
```

Comment:
I am not aware of how code licensing works, so I have decided to leave this in for now. This probably needs to be changed, though.

Reply:
Also, as @decorouz mentioned, the docstrings need to formatted according to Numpy doc syntax.